### PR TITLE
MGTK: BitBlt/rect/poly drawing routines

### PR DESCRIPTION
**Pull request by eric-ja**
Change 'char' to 'bytes' in symbols.
MGTK: Drawing routines. ($14F8E..$1584A) 
Start working on error code symbols.

### CODE REVIEW

File: mgtk.inc [hunk 1]
```diff
@@ -607,6 +607,24 @@ winfo_offset_port               := 20
 winfo_offset_nextwinfo          := 56
 winfo_size                      := 58
 
+
+;;; Errors
+
+error_in_object := $80
```
Comment (inexorabletash) [inline mgtk.inc]: There is already `inrect_inside` and `inpoly_inside` (and `*_outside`) defined above, which are used by desktop.s and some overlays (apparently I didn't update the DAs to use them, though). 

We should de-dupe. I don't have a strong preference on how, though.

File: mgtk.inc [hunk 2]
```diff
@@ -607,6 +607,24 @@ winfo_offset_port               := 20
 winfo_offset_nextwinfo          := 56
 winfo_size                      := 58
 
+
+;;; Errors
```
Comment (inexorabletash) [inline mgtk.inc]: Awesome, thanks for the initiative on this!

File: desktop/mgtk.s [hunk 3]
```diff
@@ -2135,441 +2279,558 @@ ora_2_param_bytes:
...
+.endproc
+
+
+        temp_yh        := $83
+        next_vertex    := $AA
+        current_vertex := $AC
+        loop_ctr       := $AF
+
+
+.proc process_poly
```
Comment (inexorabletash) [inline desktop/mgtk.s]: If you've got the algorithm figured out, is it worth documenting?

File: mgtk.inc (hunk 1 above)
Comment (eric-ja) [reply mgtk.inc]: Ah, yes... I didn't notice that. I'll make them the same somehow.

File: desktop/mgtk.s (hunk 3 above)
Comment (eric-ja) [reply desktop/mgtk.s]: Still a work-in-progress, but yes as it comes clearer I'll put something.

**Pull request merged**